src/components/Dialog.tsx

```diff
@@ -1,20 +1,20 @@
-'use client'
+"use client";
```

Comment:
single quotes here and everywhere else per the prettier confs - I think you should be able to use prettier to just automatically reformat according to its config - `pnpm format` should do the trick